src/index.html

```diff
@@ -25,6 +25,7 @@
     <link href="https://fonts.googleapis.com/css?family=Roboto%20Mono:400,500,700" rel="stylesheet" />
     <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet" />
     <link href="/vendor/fontello.css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://unpkg.com/ngx-bootstrap/datepicker/bs-datepicker.css">
```

Comment:
Let's vendor this

Reply:
vendored! :+1:

Reply:
Without spaces:
![image](https://user-images.githubusercontent.com/81658138/127203060-c3815a40-d2bf-4250-9375-2ecc57f24150.png)

With spaces:
![image](https://user-images.githubusercontent.com/81658138/127203175-8f3b9223-5d8d-49ad-bbe1-f9cac928a767.png)

spaces feels a bit too spaced out IMO